composer.json

```diff
@@ -142,7 +142,7 @@
         "symfony/property-access": "^5.4 || ^6.0",
         "symfony/rate-limiter": "^5.4 || ^6.0",
         "symfony/routing": "^5.4 || ^6.0",
-        "symfony/security-bundle": "^6.0.2",
+        "symfony/security-bundle": "^6.2",
```

Comment:
This is needed, because we are using `Symfony\Bundle\SecurityBundle\Security`. The new class currently extends from `Symfony\Component\Security\Core\Security` but will replace it in the future. As of Symfony 6.2 the old one is deprecated.

We should update the usage in the other places in a follow-up PR.